app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parameters = command.split(" ")
+
+    if len(parameters) == 3 \
+            and parameters[0] == "cp" \
+            and parameters[1] != parameters[2]:
```

Comment:
Use unpacking instead of using indexes to access command parts. And use `()` to split lines